fairwinds-insights.yaml

```diff
@@ -4,23 +4,23 @@ options:
 # run ./scripts/scan-all.sh to regenerate
 images:
   docker:
-    - quay.io/fairwinds/polaris:7.3
+    - quay.io/fairwinds/polaris:7.4
     - quay.io/fairwinds/nova:v3.6
-    - us-docker.pkg.dev/fairwinds-ops/oss/pluto:v5.13
-    - us-docker.pkg.dev/fairwinds-ops/oss/goldilocks:v4.6
-    - quay.io/fairwinds/insights-admission-controller:1.9.5
-    - quay.io/fairwinds/aws-costs:1.1.2
-    - quay.io/fairwinds/insights-ci:4.2.8
-    - quay.io/fairwinds/falco-agent:0.2.6
-    - quay.io/fairwinds/fw-kube-bench-aggregator:0.3.7
-    - quay.io/fairwinds/fw-kube-bench:0.4.7
-    - quay.io/fairwinds/kubectl:0.20.2
-    - quay.io/fairwinds/fw-kubesec:1.4.2
-    - quay.io/fairwinds/fw-opa:2.2.6
-    - quay.io/fairwinds/prometheus-collector:1.1.8
-    - quay.io/fairwinds/rbac-reporter:1.3.6
-    - quay.io/fairwinds/right-sizer:0.4.6
-    - quay.io/fairwinds/fw-trivy:0.24.8
-    - quay.io/fairwinds/insights-uploader:0.4.2
-    - quay.io/fairwinds/insights-utils:0.0.3
-    - quay.io/fairwinds/workloads:2.4.4
+    - us-docker.pkg.dev/fairwinds-ops/oss/pluto:v5.16
+    - us-docker.pkg.dev/fairwinds-ops/oss/goldilocks:v4.8
+    - quay.io/fairwinds/insights-admission-controller:1.9.9
+    - quay.io/fairwinds/aws-costs:1.3.0
+    - quay.io/fairwinds/insights-ci:5.0.4
```

Comment:
based on the insights report, looks like `insights-ci:5.0.4` does not yet exist, so you can use `5.0.3` here

Reply:
That was one reason for my aborted attempt to change the security scanning step a bit in this pipeline.  My understanding right now is that `5.0.4` will be released once this PR is merged into `main`, so we might as well have trivy scanning the latest version of this, until we change how security scanning behaves for branches and PRs.